README.md

```diff
@@ -320,6 +322,38 @@ So it can be used with our [Docker Build Push action](https://github.com/docker/
 |-------------------------------|------|------------------------------------------------------------------------------------------------------------|
 | `DOCKER_METADATA_PR_HEAD_SHA` | Bool | If `true`, set associated head SHA instead of commit SHA that triggered the workflow on pull request event |
 
+## `context` input
+
+`context` defines where to get Git context data. Allowed options are "workflow"  (default), "git".
+
+
+- `workflow` context
+
+Will get Git context from workflow context. See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
+
+
+```yaml
+context: workflow
+```
+
+```
+sha: context.sha
+ref: context.ref
+```
+
+- `git` context
+
+Will get Git context from previously checkout repository. See https://github.com/actions/checkout
+
+```yaml
+context: git
+```
+
+```
+sha: git rev rev-parse HEAD
+ref: git symbolic-ref HEAD
+```
+
```

Comment:
I don't think we need that level of detail. Would also prefer the same layout as other inputs. Something like this should enough:

<pre>
## `context` input

`context` defines where to get context metadata:

```yaml
# default
context: workflow
# or
context: git
```

* `workflow`: Get context metadata from the workflow (GitHub context). See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
* `git`: Get context metadata from the workflow and overrides some of them with current Git context, such as `ref` and `sha`.
</pre>